docs/source/deduplication.rst

```diff
@@ -0,0 +1,42 @@
+System Deduplication and Merging
```

Comment:
Global: I will have a few comments below about changing verb tenses. In general, stick to the simplest tense possible and avoid the "fancy" ones that attempt to split hairs about the action's place in space/time.